packages/devtools/devtools-extension/vite.config.ts

```diff
@@ -0,0 +1,111 @@
+//
+// Copyright 2022 DXOS.org
+//
+
+import ReactPlugin from '@vitejs/plugin-react';
+import { resolve } from 'node:path';
+import { defineConfig } from 'vite';
+import { VitePluginFonts } from 'vite-plugin-fonts';
+import { crx as chromeExtensionPlugin } from '@crxjs/vite-plugin';
+
+import { ConfigPlugin } from '@dxos/config/vite-plugin';
+import { ThemePlugin } from '@dxos/react-components/plugin';
+import { kaiThemeExtension } from '@dxos/kai/theme-extensions';
+import { osThemeExtension } from '@dxos/react-ui/theme-extensions';
+
+import manifest from './manifest.json';
+
+const env = (value?: string) => (value ? `"${value}"` : undefined);
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  base: '', // Ensures relative path to assets.
+  optimizeDeps: {
+    force: true,
+    include: [
+      '@dxos/async',
+      '@dxos/client',
+      '@dxos/client-services',
+      '@dxos/codec-protobuf',
+      '@dxos/config',
+      '@dxos/credentials',
+      '@dxos/debug',
+      '@dxos/devtools',
+      '@dxos/kai',
+      '@dxos/log',
+      '@dxos/network-manager',
+      '@dxos/protocols',
+      '@dxos/react-async',
+      '@dxos/react-client',
+      '@dxos/react-components',
+      '@dxos/react-toolkit',
+      '@dxos/react-ui',
+      '@dxos/rpc'
+    ]
+  },
+  build: {
+    commonjsOptions: {
+      include: [/packages/, /node_modules/]
+    },
+    rollupOptions: {
+      input: {
+        panel: resolve(__dirname, 'panel.html')
```

Comment:
other html files are also needed I assume?

Reply:
Other files are mentioned in the manifest and vite extension plugin handles them

Reply:
in this case, is it necessary to import the manifest config from `manifest.json`? could you just have it as an object inline?

Reply:
We can do inline object